src/dcos_e2e/backends/_docker/_containers.py

```diff
@@ -86,6 +86,7 @@ def start_dcos_container(
     docker_storage_driver: DockerStorageDriver,
     docker_version: DockerVersion,
     network: Optional[docker.models.networks.Network] = None,
+    ports: Dict[str, Any] = {},
```

Comment:
Avoid using dictionary defaults in Python. Dictionaries are mutable, so different function calls can interact, e.g.:

```python
import random

def example(param={}):
	random = random.randint(1, 10)
	param[random] = random
	print(param)

example()
example()
```

can print:

```
{7: 7}
{7: 7, 8: 8}
```

Instead, use:

```python
def foo(
    ports: Optional[Dict[str, Any]] = None,
) -> None:
    ports = ports or {}
```

Reply:
Ah, thanks! Will fix it. My python experience is almost zero :)